### PR TITLE
Enrollment tile updates & refactor

### PR DESCRIPTION
**Pull request by scaustin34**
IVL-183420120

Wholesale rewrite of the plan/enrollment tiles to a more consistent and stable styling. Various stylistic changes per ME. Retained all the DC/Shop functionality should it be pitched to them, though they will not receive this update currently.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -481,7 +481,7 @@ def carrier_logo(plan)
 
   def display_carrier_logo(plan, options = {:width => 50})
     carrier_name = carrier_logo(plan)
-    image_tag("logo/carrier/#{carrier_name.parameterize.underscore}.jpg", width: options[:width]) # Displays carrier logo (Delta Dental => delta_dental.jpg)
+    image_tag("logo/carrier/#{carrier_name.parameterize.underscore}.jpg", width: options[:width], alt: "#{carrier_name} logo") # Displays carrier logo (Delta Dental => delta_dental.jpg)
```
Comment (polographer) [inline app/helpers/application_helper.rb:484]: thank you

File: app/models/hbx_enrollment.rb [hunk 2]
```diff
@@ -1877,6 +1877,13 @@ def covered_members_first_names
     end
   end
 
+  def covered_members_name_age
+    enrollment_members = hbx_enrollment_members.sort_by { |a| a.is_subscriber ? 0 : 1 }
+    enrollment_members.inject([]) do |name_age, member|
+      name_age << "#{member.person.first_name} (#{((Time.zone.now - member.person.dob.to_time) / 1.year.seconds).floor})"
```
Comment (polographer) [inline app/models/hbx_enrollment.rb:1883]: is this formmated for screen ? if yes, could this be on a view helper ? if no, would it be more reusable to return it on a hash ?

File: app/views/insured/families/_enrollment_actions.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            <button class="btn btn-default dropdown-toggle interaction-click-control-actions" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                <%= l10n("actions")%>
+                <span class="caret"></span>
+            </button>
+
+            <div class="dropdown-menu dropdown-glossary run-glossary" role="menu" aria-labelledby="dropdownmenuButton">
+                <div class="dropdown-divider" style="border-top: 1px solid #e9ecef"></div>
+
+                <a href="<%= summary_products_plans_path({standard_component_id: hbx_enrollment.product.hios_id, hbx_enrollment_id: hbx_enrollment.id, active_year: hbx_enrollment.product.active_year, source: 'account'}) %>" class='btn-link btn-block dropdown-item' id='view-details-btn' style='padding: 6px 12px; margin: 4px 0;' data-remote='true'>
+                    <%= render partial:"shared/glossary_hover", locals: {key: "view_coverage_details", title: "View my coverage details", term: sanitize(l10n("view_details")) } %>
```
Comment (polographer) [inline app/views/insured/families/_enrollment_actions.html.erb:15]: should the title be i10n ?

File: app/models/hbx_enrollment.rb (hunk 2 above)
Comment (scaustin34) [reply app/models/hbx_enrollment.rb:1883]: moved to families_helper

**Review by polographer — APPROVED** [review]
looks good!

**Pull request merged**